docs/writing-docs/autodocs.md

```diff
@@ -98,25 +123,44 @@ Storybook provides support for MDX 2, an open standard markup language, combinin
 
 <!-- prettier-ignore-end -->
 
-Create an MDX file for your component in the same directory as your stories and components to add your custom documentation. We recommend naming the file similar to your component (e.g., `Button.mdx`) for easy reference.
+Create an MDX file to add your custom documentation. Depending on how you want your documentation to render in the UI, you'll need to consider the following use cases.
+
+### Using the `Meta` Doc Block
+
+If you need to match the component documentation to an existing story, you can use the `Meta` Doc Block and configure it to control how the documentation gets rendered. Out of the box, it allows you to define a custom title or a reference to the story you need to document (i.e., via the `of` prop). For example:
 
 <!-- prettier-ignore-start -->
 
 <CodeSnippets
   paths={[
-    'common/storybook-auto-docs-baseline-example.mdx.mdx',
+    'common/storybook-auto-docs-baseline-example.custom-title.mdx.mdx',
+    'common/storybook-auto-docs-baseline-example.of-prop.mdx.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+### Using the File System
+
+However, providing the `Meta` Doc Block may not be required for certain use cases, such as standalone pages or even as guidelines for testing your components. In that case, you can safely omit it. Storybook will instead rely on the file's physical location to place the documentation in the sidebar, overriding any pre-existent auto-generated documentation with your own. For example:
```

Comment:
Could we reference and leverage the [existing autotitle docs](https://storybook.js.org/docs/7.0/react/configure/sidebar-and-urls#csf-30-auto-titles) when explaining this?

Reply:
Addressed with the latest commit. Marking this as solved.